plugins/kubernetes-backend/src/service/KubernetesProxy.ts

```diff
@@ -34,7 +41,24 @@ export const APPLICATION_JSON: string = 'application/json';
  *
  * @public
  */
-export const HEADER_KUBERNETES_CLUSTER: string = 'X-Kubernetes-Cluster';
+export const HEADER_KUBERNETES_CLUSTER: string = 'Backstage-Kubernetes-Cluster';
+
+/**
+ * The header that is used to specify the Authentication Authorities token.
+ * e.x if using the google auth provider as your authentication authority then this field would be the google provided bearer token.
+ * @public
+ */
+export const HEADER_KUBERNETES_AUTH: string =
+  'Backstage-Kubernetes-Authorization';
+
+/**
+ * The options object expected to be passed as a parameter to KubernetesProxy.createRequestHandler().
+ *
+ * @public
+ */
+export type createRequestHandlerOptions = {
```

Comment:
Types should should use PascalCase, so uppercase `C` here. While we're at it I'm thinking this is perhaps a bit vague, feeling we could get the proxy word in there too. Perhaps `CreateProxyRequestHandlerOptions` instead, or we go full verbose with `KubernetesProxyCreateRequestHandlerOptions`.

Reply:
Sure thing I will change this. I think I like the full verbose name more so i will make that change.

Reply:
Done. Changed to full verbose naming.